core-bundle/tests/EventListener/DataContainer/PageRoutingListenerTest.php

```diff
@@ -38,10 +42,17 @@ public function testGetsPathFromPageRoute(): void
         $framework = $this->mockContaoFramework([PageModel::class => $pageAdapter]);
 
         $pageRoute = $this->createMock(PageRoute::class);
+
```

Comment:
This seems to be an unrelated change that also does not match the code style.

Reply:
so we're discussing empty newline codestyle now? To me it's a newline because we have more than one call on the object. But feel free to CS it 🤷

Reply:
No, we are just discussing unrelated changes. You should make them in a separate PR. 🤷‍♂️